terminal/command.go

```diff
@@ -40,25 +40,28 @@ type Commands struct {
 	client  service.Client
 }
 
+var c *Commands
+
 // Returns a Commands struct with default commands defined.
 func DebugCommands(client service.Client) *Commands {
-	c := &Commands{client: client}
+	c = &Commands{client: client}
 
 	c.cmds = []command{
-		{aliases: []string{"help"}, cmdFn: c.help, helpMsg: "Prints the help message."},
+		{aliases: []string{"help", "h"}, cmdFn: c.help, helpMsg: "Prints the help message."},
 		{aliases: []string{"break", "b"}, cmdFn: breakpoint, helpMsg: "Set break point at the entry point of a function, or at a specific file/line. Example: break foo.go:13"},
 		{aliases: []string{"continue", "c"}, cmdFn: cont, helpMsg: "Run until breakpoint or program termination."},
 		{aliases: []string{"step", "si"}, cmdFn: step, helpMsg: "Single step through program."},
 		{aliases: []string{"next", "n"}, cmdFn: next, helpMsg: "Step over to next source line."},
-		{aliases: []string{"threads"}, cmdFn: threads, helpMsg: "Print out info for every traced thread."},
+		{aliases: []string{"threads", "allm"}, cmdFn: threads, helpMsg: "Print out info for every traced thread."},
```

Comment:
Not necessarily on board with this alias, a casual user would likely not know that internally the scheduler refers to threads as Ms.